Package.swift

```diff
@@ -24,7 +24,7 @@ func pseudoPkgConfigText(_ package: String) -> String? {
   guard let pcp = ProcessInfo.processInfo.environment["PKG_CONFIG_PATH"] else { return nil }
 
   return pcp.split(separator: pathSeparator)
-    .lazy.compactMap({ try? String(contentsOfFile: "\($0)/\(package).pc") }).first
+    .lazy.compactMap({ try? String(contentsOfFile: "\($0)/\(package).pc", encoding: String.Encoding.utf8) }).first
```

Comment:
We wrote the script that generates this file so you don't have to pray.  `pkgconfig` is really old, crude technology last time I looked, so I bet it only handles ASCII correctly anyway.